apps/services/search-indexer/bin/generate-es-config.sh

```diff
@@ -49,11 +49,11 @@ replace_full() {
 }
 
 main() {
-	local lang="$1"
+	# local lang="$1"
```

Comment:
leaving the comment in there because of the argument order? maybe add comment for future context?

Reply:
unused first variable. not wanting to change the invocations, this was the simplest way. IMO the comment speaks for itself...